src/Bridge/Symfony/Resources/config/services.php

```diff
@@ -57,7 +57,7 @@
             ->args([
                 '$client' => service(Client::class),
                 '$serializer' => service('serializer'),
-                '$bulkMaxSize' => 100,
+                '$bulkMaxSize' => abstract_arg('bulk size'),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is it normal there is no underscore in the name here? 'bulk size'

Reply:
It is, because it's only a description, not a name or so.